### PR TITLE
ux docs

### PR DESCRIPTION
**Pull request by Blackbaud-ToddRoberts**

### CODE REVIEW

File: src/app/docs/text-editor/text-editor-docs.component.html [hunk 1]
```diff
@@ -20,5 +20,120 @@
...
+          <ul>
+            <li>Merge</li>
+          </ul>
+          <p class="sky-docs-text-block">
+            Default toolbar items:
+          </p>
+          <ul>
+            <li>Font</li>
+            <li>Text size</li>
+            <li>Text styles (bold/italic/underline)</li>
```
Comment (blackbaud-johnly) [inline src/app/docs/text-editor/text-editor-docs.component.html:101]:
```suggestion
            <li>Text style (bold/italic/underline)</li>
```

File: src/app/docs/text-editor/text-editor-docs.component.html [hunk 2]
```diff
@@ -20,5 +20,120 @@
...
+            <li>Text color</li>
+            <li>Highlight color</li>
+            <li>Lists (numbered/bulleted)</li>
```
Comment (blackbaud-johnly) [inline src/app/docs/text-editor/text-editor-docs.component.html:104]:
```suggestion
            <li>List format (numbered/bulleted)</li>
```

File: src/app/docs/text-editor/text-editor-docs.component.html [hunk 3]
```diff
@@ -20,5 +20,120 @@
...
+            <li>Hyperlink</li>
+          </ul>
+          <p class="sky-docs-text-block">
+            Optional toolbar items:
```
Comment (blackbaud-johnly) [inline src/app/docs/text-editor/text-editor-docs.component.html:108]:
```suggestion
            Optional toolbar options:
```
I don't love "Optional ... options," but I think "option" is less generic than "item" and aligns with the distinction in that terminology that Adam and I have discussed.

File: src/app/docs/text-editor/text-editor-docs.component.html [hunk 4]
```diff
@@ -20,5 +20,120 @@
...
+            caption="Text editor with default options."
+            imageSource="~/assets/img/option-menu-toolbar-items.png"
+          >
+          </sky-docs-design-guideline-thumbnail>
+        </sky-docs-design-guideline>
+      </sky-docs-demo-page-section>
+
+      <sky-docs-demo-page-section
+      heading="Related information"
+    >
```
Comment (blackbaud-johnly) [inline src/app/docs/text-editor/text-editor-docs.component.html:125]: Is it worth adding a Components section here to link to https://developer.blackbaud.com/skyux/components/form? And what about https://developer.blackbaud.com/skyux-forms/docs/input-box?docs-active-tab=design? I'm not sure either way, but just throwing it out to see if those are related docs that consumers should be aware of.

File: src/app/docs/text-editor/text-editor-docs.component.html [hunk 5]
```diff
@@ -20,5 +20,120 @@
...
+        heading="Usage"
+      >
+        <sky-docs-design-guideline
+          heading="Use when"
+          headingStyle="success"
+        >
+          <p>
+            Use a text editor when users need to create blocks of text that include formatting, hyperlinks or merge fields within the text.
+          </p>
+          <sky-docs-design-guideline-thumbnail
```
Comment (blackbaud-johnly) [inline src/app/docs/text-editor/text-editor-docs.component.html:36]: Do we need two images here? One for default theme and one for modern theme? Adam's been including two images, so I'm assuming we need that here as well?

File: src/app/docs/text-editor/text-editor-docs.component.html (hunk 3 above)
Comment (Blackbaud-ToddRoberts) [reply src/app/docs/text-editor/text-editor-docs.component.html:108]: What do you think about "actions" or "buttons"?

Comment (blackbaud-johnly) [reply src/app/docs/text-editor/text-editor-docs.component.html:108]: I think `actions` works. Not as sure about `buttons`.

File: src/app/docs/text-editor/text-editor-docs.component.html (hunk 5 above)
Comment (Blackbaud-ToddRoberts) [reply src/app/docs/text-editor/text-editor-docs.component.html:36]: updated

File: src/app/docs/text-editor/text-editor-docs.component.html [hunk 6]
```diff
@@ -20,5 +20,120 @@
...
+              Label
+            </sky-docs-anatomy-item>
+            <sky-docs-anatomy-item>
+              Menu bar
+            </sky-docs-anatomy-item>
+            <sky-docs-anatomy-item>
+              Toolbar
+            </sky-docs-anatomy-item>
+            <sky-docs-anatomy-item>
+              Text entry field
```
[the remark this answers is not in the document]
Comment (Blackbaud-ToddRoberts) [reply src/app/docs/text-editor/text-editor-docs.component.html]: updated

File: src/app/docs/text-editor/text-editor-docs.component.html [hunk 7]
```diff
@@ -20,5 +20,120 @@
...
+          >
+          </sky-docs-design-guideline-thumbnail>
+        </sky-docs-design-guideline>
+  
+        <sky-docs-design-guideline
+          heading="Don't use when"
+          headingStyle="danger"
+        >
+          <p class="sky-docs-text-block">
+            Don’t use the text editor when users only need to enter unformatted plain text. Use a text input or textarea instead.
```
[the remark this answers is not in the document]
Comment (Blackbaud-ToddRoberts) [reply src/app/docs/text-editor/text-editor-docs.component.html]: updated

File: src/app/docs/text-editor/text-editor-docs.component.html (hunk 3 above)
Comment (Blackbaud-ToddRoberts) [reply src/app/docs/text-editor/text-editor-docs.component.html]: updated

File: src/app/docs/text-editor/text-editor-docs.component.html [hunk 8]
```diff
@@ -20,5 +20,120 @@
...
+            <li>Format</li>
+          </ul>
+          <p class="sky-docs-text-block">
+            Optional menu item:
+          </p>
+          <ul>
+            <li>Merge</li>
+          </ul>
+          <p class="sky-docs-text-block">
+            Default toolbar items:
```
[the remark this answers is not in the document]
Comment (Blackbaud-ToddRoberts) [reply src/app/docs/text-editor/text-editor-docs.component.html]: updated

File: src/app/docs/text-editor/text-editor-docs.component.html (hunk 4 above)
Comment (Blackbaud-ToddRoberts) [reply src/app/docs/text-editor/text-editor-docs.component.html:158]: i don't think we need related components - the text editor will be pretty different than other inputs

**Review by blackbaud-johnly — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-text-editor?_sp.spa=pr-ux-docs](https://host.nxt.blackbaud.com/skyux-text-editor?_sp.spa=pr-ux-docs)

**Review by Blackbaud-AdamFunderburk — APPROVED** [review]

**Pull request merged**